packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -131,54 +161,98 @@ export const MAINNET_CURVE_INFOS: { [name: string]: CurveInfo } = {
     },
-    // Looks like it's dying.
-    DaiUsdcUsdtBusd: {
+    BUSD: {
         exchangeFunctionSelector: CurveFunctionSelectors.exchange_underlying,
         sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy_underlying,
         buyQuoteFunctionSelector: CurveFunctionSelectors.get_dx_underlying,
-        poolAddress: '0x79a8c46dea5ada233abaffd40f3a0a2b1e5a4f27',
-        tokens: [
-            TOKENS.DAI,
-            TOKENS.USDC,
-            TOKENS.USDT,
-            '0x4fabb145d64652a948d72533023f6e7a623c7c53', // bUSD
-        ],
+        poolAddress: POOLS.curve_BUSD,
+        tokens: [TOKENS.DAI, TOKENS.USDC, TOKENS.USDT, TOKENS.BUSD],
     },
-    DaiUsdcUsdtSusd: {
+    sUSD: {
         exchangeFunctionSelector: CurveFunctionSelectors.exchange_underlying,
         sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy_underlying,
         buyQuoteFunctionSelector: CurveFunctionSelectors.None,
-        poolAddress: '0xa5407eae9ba41422680e2e00537571bcc53efbfd',
+        poolAddress: POOLS.curve_sUSD,
         tokens: [TOKENS.DAI, TOKENS.USDC, TOKENS.USDT, TOKENS.sUSD],
     },
-    RenbtcWbtc: {
+    renBTC: {
         exchangeFunctionSelector: CurveFunctionSelectors.exchange,
         sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy,
         buyQuoteFunctionSelector: CurveFunctionSelectors.None,
-        poolAddress: '0x93054188d876f558f4a66b2ef1d97d16edf0895b',
+        poolAddress: POOLS.curve_renBTC,
         tokens: [TOKENS.RenBTC, TOKENS.WBTC],
     },
-    RenbtcWbtcSbtc: {
+    sBTC: {
         exchangeFunctionSelector: CurveFunctionSelectors.exchange,
         sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy,
         buyQuoteFunctionSelector: CurveFunctionSelectors.None,
-        poolAddress: '0x7fc77b5c7614e1533320ea6ddc2eb61fa00a9714',
+        poolAddress: POOLS.curve_sBTC,
         tokens: [TOKENS.RenBTC, TOKENS.WBTC, TOKENS.sBTC],
     },
-    TriPool: {
+    HBTC: {
         exchangeFunctionSelector: CurveFunctionSelectors.exchange,
         sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy,
         buyQuoteFunctionSelector: CurveFunctionSelectors.None,
-        poolAddress: '0xbebc44782c7db0a1a60cb6fe97d0b483032ff1c7',
+        poolAddress: POOLS.curve_HBTC,
+        tokens: [TOKENS.hBTC, TOKENS.WBTC],
+    },
+    TRI: {
+        exchangeFunctionSelector: CurveFunctionSelectors.exchange,
+        sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy,
+        buyQuoteFunctionSelector: CurveFunctionSelectors.None,
+        poolAddress: POOLS.curve_TRI,
         tokens: [TOKENS.DAI, TOKENS.USDC, TOKENS.USDT],
     },
+    // Metapools
+    GUSD: {
+        exchangeFunctionSelector: CurveFunctionSelectors.exchange_underlying,
+        sellQuoteFunctionSelector: CurveFunctionSelectors.get_dy_underlying,
+        buyQuoteFunctionSelector: CurveFunctionSelectors.None,
+        poolAddress: POOLS.curve_GUSD,
+        tokens: [TOKENS.GUSD, TOKENS.DAI, TOKENS.USDC, TOKENS.USDT],
```

Comment:
Do these pools clash with the TriPool? 

In the case of a DAI/USDC sell would the GUSD pool and the TriPool both return samples and potentially result in trading against the same underlying (TriPool)?

Or does it not work like that, and the trade has to include GUSD?

Reply:
good point, let me try to run a few tests

Reply:
✅